tests/reference/asr-test_list2-10ffdd7.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: append() takes exactly one argument
+ --> tests/errors/test_list2.py:6:5
+  |
+6 |     a.append(4, 5)
+  |     ^^^^^^^^^^^^^^ 
```

Comment:
In the future, we can improve these messages, here is an idea:
```suggestion
6 |     a.append(4, 5)
  |       ^^^^^^ ┬  ┬                       
  |          │   └──┴── two arguments provided    
  |          │                              
  |        expected one argument       
```

This is how it looks like in a terminal (without colors):

<img width="331" alt="Screen Shot 2022-04-22 at 7 46 34 AM" src="https://user-images.githubusercontent.com/20568/164727038-bffbdf94-1f5c-4c31-bcb7-4b765b45316d.png">

And of course there would be colors.